src/cmd/tools/dtest/docker/harness/resources/common.go

```diff
@@ -51,6 +51,8 @@ type dockerResourceOptions struct {
 	overrideDefaults bool
 	source           string
 	containerName    string
+	imageName        string
+	imageTag         string
```

Comment:
Nit: I think things might clean up a little if you used the `dockerImage` struct here instead of these 2 fields that seem to be accessed together everyehere.